apps/researcher/src/app/[locale]/objects/settings-menu.tsx

```diff
@@ -0,0 +1,114 @@
+'use client';
+
+import PopoverMenu from '@/components/popover';
+import {useListStore} from '@colonial-collections/list-store';
+import {ListBulletIcon, Squares2X2Icon} from '@heroicons/react/24/solid';
+import {useTranslations} from 'next-intl';
+import {ImageVisibility} from './definitions';
+
+export default function SettingsMenu() {
+  const t = useTranslations('Settings');
+  const view = useListStore(s => s.view);
+  const imageVisibility = useListStore(s => s.imageVisibility);
+  const viewChange = useListStore(s => s.viewChange);
+  const imageVisibilityChange = useListStore(s => s.imageVisibilityChange);
+  const limitChange = useListStore(s => s.limitChange);
+  const limit = useListStore(s => s.limit);
+
+  return (
+    <PopoverMenu buttonText={t('settings')} variant="default">
+      <div className="flex w-[300px] flex-col">
+        <div className="p-4 border-b flex gap-2 flex-wrap">
+          <div className="w-full text-sm italic">{t('resultView')}:</div>
+          <button
+            onClick={() => viewChange('grid')}
+            className="p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-neutral-200/50 hover:bg-neutral-300/50 text-neutral-800 transition flex items-center gap-1"
+          >
+            <Squares2X2Icon className="w-4 h-4 fill-neutral-800" />
+            {t('grid')}
+          </button>
+          <button
+            onClick={() => viewChange('list')}
+            className="p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-neutral-200/50 hover:bg-neutral-300/50 text-neutral-800 transition flex items-center gap-1"
+          >
+            <ListBulletIcon className="w-4 h-4 fill-neutral-800" />
+            {t('list')}
+          </button>
+        </div>
+        <div className="p-4 border-b flex gap-2 flex-wrap">
+          <div className="w-full text-sm italic">{t('resultAmount')}</div>
```

Comment:
About `resultAmount` - I think that `number` is used for quantities that can be counted (and 'amount' for uncountable). So: `resultNumber` (or `resultCount` or `numberOfResults` or `resultLimit`)?

Reply:
Because the property name is limit, I will go for `resultLimit`